OneLogin/functions/Set-OneLoginUserPasswd.ps1

```diff
@@ -0,0 +1,43 @@
+function Set-OneLoginUserPasswd

```

Comment:
Please use a more explicit noun name for this, such as `Set-OneLoginUserPassword`.

Reply:
That shouldn't be an issue.